### PR TITLE
update(cassandra): add ZDM_ORIGIN_ENABLE_HOST_ASSIGNMENT=false

### PR DESCRIPTION
**Pull request by wojcik-dorota**
https://aiven.atlassian.net/browse/DOC-915

## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](../styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

**Comment by jorgeuribe-aiven** [comment]
This doc needs to be updated, it is still using ZDM_SOURCE_* for all the env variables, I assume naming has changed in newer versions, see:  
https://github.com/datastax/zdm-proxy?tab=readme-ov-file#quick-start

the ZDM_ORIGIN_ENABLE_HOST_ASSIGNMENT=false. is already mentioned on this doc but it is under the old naming: ZDM_SOURCE_ENABLE_HOST_ASSIGNMENT=false

let me know if you need help to make the other changes or if you want me to take this PR from here.

**Comment by jorgeuribe-aiven** [comment]
@wojcik-dorota please review the changes I have pushed.

File: docs/products/cassandra/howto/zdm-proxy.md [hunk 1]
```diff
@@ -78,14 +78,14 @@ LICENSE  zdm-proxy-linux-amd64-v2.1.0.tgz  zdm-proxy-v2.1.0
...
-environment variables using the `export` command. Next, run the binary.
+To run ZDM Proxy, specify connection information by setting `ZDM_TARGET_*` and `ZDM_ORIGIN_*` 
+environment variables using the `export` command, **ORIGIN** refers to the SOURCE service. Next, run the binary.
 
 ```bash
-export ZDM_SOURCE_CONTACT_POINTS=localhost
-export ZDM_SOURCE_USERNAME=cassandra
-export ZDM_SOURCE_PASSWORD=cassandra
-export ZDM_SOURCE_PORT=1234
+export ZDM_ORIGIN_CONTACT_POINTS=localhost
```
Comment (jorgeuribe-aiven) [inline docs/products/cassandra/howto/zdm-proxy.md:85]: Changed the SOURCE variable to ORIGIN as per docs.

File: docs/products/cassandra/howto/zdm-proxy.md [hunk 2]
```diff
@@ -134,7 +135,7 @@ The port that ZDM Proxy uses is 14002, which can be overridden.
 1.  Check data in the table.
 
     ```bash
-    select * from TABLE_NAME.DATABASE_NAME;
+    select * from KEYSPACE_NAME.TABLE_NAME;
```
Comment (jorgeuribe-aiven) [inline docs/products/cassandra/howto/zdm-proxy.md:138]: Cassandra format should be keyspace_name.table_name

**Review by jorgeuribe-aiven — COMMENTED** [review]
I have made changes, let me know if any change is needed.

**Review by wojcik-dorota — COMMENTED** [review]
Thanks :-) We'll take it from here: review and merge soon!

**Review by ArthurFlag — APPROVED** [review]

**Pull request merged**